libcontainerd/remote/client.go

```diff
@@ -220,9 +220,9 @@ func (c *container) Start(ctx context.Context, checkpointDir string, withStdin b
 
 	t, err = c.c8dCtr.NewTask(ctx,
 		func(id string) (cio.IO, error) {
-			fifos := newFIFOSet(bundle, id, withStdin, spec.Process.Terminal)
+			fifos := newFIFOSet(ctx, bundle, id, withStdin, spec.Process.Terminal)
 
-			rio, err = c.createIO(fifos, stdinCloseSync, attachStdio)
+			rio, err = c.createIO(ctx, fifos, stdinCloseSync, attachStdio)
```

Comment:
Was wondering how about these two.

Looked at the current implementation and this is called right in `NewTask`: https://github.com/containerd/containerd/blob/40f26543bdc27cbe8b058ac082e91c5832bb1c41/container.go#L216

Looking at the usage, it doesn't seem like this would change, so I think it's fine.